tests/unit/dojson/test_dojson_hep_bd01x09x.py

```diff
@@ -243,7 +243,7 @@ def test_persistent_identifiers_from_0247_a_2_corrects_to_hdl():
     expected = [
```

Comment:
this whole test is broken,  `0247_` with `2` = `DOI` should go to [dois](https://github.com/inspirehep/inspire-schemas/blob/master/inspire_schemas/records/hep.json#L318).
`0247_` with `2` = `HDL` (e.g. [record/1501141](https://inspirehep.net/record/1501141/export/xm)) goes to `persistent_identifiers`.